src/PKSim.Presentation/Presenters/Main/StatusBarPresenter.cs

```diff
@@ -142,26 +149,26 @@ public void Handle(ProgressInitEvent eventToHandle)
             .And.Visible(true);
 
          update(StatusBarElements.ProgressStatus)
-            .WithCaption($"{_runningSimulationsDictionary.Count} {eventToHandle.Message}")
+            .WithCaption($"{_interactiveSimulationRunner.ActiveSimulationsCount} {eventToHandle.Message}")
             .And.Visible(true);
       }
 
       public void Handle(SimulationRunCanceledEvent eventToHandle)
       {
-         _runningSimulationsDictionary[eventToHandle.Simulation.Id] = false;
-         if (activeSimulationsCount == 0)
+         if (_interactiveSimulationRunner.ActiveSimulationsCount == 0)
          {
             resetCountersAndHideBar();
          }
       }
 
       public void Handle(ProgressingEvent eventToHandle)
       {
+         var message = _interactiveSimulationRunner.ActiveSimulationsCount == 0 ? string.Empty : _interactiveSimulationRunner.ActiveSimulationsCount.ToString();
          update(StatusBarElements.ProgressBar)
             .WithValue(eventToHandle.ProgressPercent);
 
          update(StatusBarElements.ProgressStatus)
-            .WithCaption($"{activeSimulationsCount} {eventToHandle.Message}");
+            .WithCaption($"{message} {eventToHandle.Message}");
```

Comment:
same issue here. It will show 1 and a message if only one simulation. Or do I see this wrong?